contracts/BSCValidatorSet.sol

```diff
@@ -551,11 +551,9 @@ contract BSCValidatorSet is IBSCValidatorSet, System, IParamSubscriber, IApplica
     uint256 totalValue;
     uint256 balanceOfSystemReward = address(SYSTEM_REWARD_ADDR).balance;
     if (balanceOfSystemReward > MAX_SYSTEM_REWARD_BALANCE) {
-      totalValue = balanceOfSystemReward.div(100);
-    } else if (balanceOfSystemReward > previousBalanceOfSystemReward) {
       // when a slash happens, theres will no rewards in some epoches,
       // it's tolerated because slash happens rarely
-      totalValue = (balanceOfSystemReward.sub(previousBalanceOfSystemReward)).mul(finalityRewardRatio).div(100);
+      totalValue = balanceOfSystemReward.sub(MAX_SYSTEM_REWARD_BALANCE);
```

Comment:
if this is the way, then `previousBalanceOfSystemReward`. is useless.

Reply:
> if this is the way, then `previousBalanceOfSystemReward`. is useless.

yes useless

Reply:
Will this change be backwards compatible with historical blocks?